src/cryptonote_core/blockchain.cpp

```diff
@@ -5430,7 +5430,7 @@ void Blockchain::cancel()
 }
 
 #if defined(PER_BLOCK_CHECKPOINT)
-static const char expected_block_hashes_hash[] = "ff99bc76e59e0d6268e2d8ed54a2206d828de643fcb13be8d296611fc051a540";
+static const char expected_block_hashes_hash[] = "134194090ed578843d24734e766760a50463961200d8e8a9dda410c1c18e30db";
```

Comment:
Matches my output of the command:
`./monero-blockchain-export --output-file checkpoints.dat --block-stop 2478000 --blocksdat && sha256sum checkpoints.dat`

Reply:
Also checked, that the file `src/blocks/checkpoints.dat` has the same hash, with:
`sha256sum src/blocks/checkpoints.dat`